app/frontend/src/features/communities/discussions/DiscussionPage.tsx

```diff
@@ -0,0 +1,130 @@
+import { Typography } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+import Alert from "components/Alert";
+import Avatar from "components/Avatar";
+import CircularProgress from "components/CircularProgress";
+import Divider from "components/Divider";
+import HeaderButton from "components/HeaderButton";
+import { BackIcon } from "components/Icons";
+import Markdown from "components/Markdown";
+import PageTitle from "components/PageTitle";
+import { useUser } from "features/userQueries/useUsers";
+import { Error as GrpcError } from "grpc-web";
+import { Discussion } from "pb/discussions_pb";
+import { discussionKey } from "queryKeys";
+import { useQuery } from "react-query";
+import { useHistory, useParams } from "react-router-dom";
+import { service } from "service";
+import { dateFormatter, timestamp2Date } from "utils/date";
+import makeStyles from "utils/makeStyles";
+
+import { ADDED_BY, PREVIOUS_PAGE, UNKNOWN_USER } from "../constants";
+import CommentTree from "./CommentTree";
+
+const useStyles = makeStyles((theme) => ({
+  header: {
+    alignItems: "center",
+    display: "flex",
+  },
+  discussionTitle: {
+    marginInlineStart: theme.spacing(2),
+  },
+  discussionContent: {
+    margin: 0,
+  },
+  addedByLabel: {
+    marginBlockStart: theme.spacing(2),
+    marginBlockEnd: theme.spacing(1),
+  },
+  creatorContainer: {
+    "& > * + *": {
+      marginInlineStart: theme.spacing(2),
+    },
+    alignItems: "center",
+    display: "flex",
+    marginBlockEnd: theme.spacing(2),
+  },
+  creatorDetailsContainer: {
+    display: "flex",
+    flexDirection: "column",
+  },
+  avatar: {
+    height: "3rem",
+    width: "3rem",
+  },
+}));
+
+export const CREATOR_LOADING_TEST_ID = "creator-loading-state";
+
+export default function DiscussionPage() {
```

Comment:
It would be great if this could be shown within the community page - makes it less bare, and discussions are always part of a community.

Maybe that means making the route `/community/1/world/discussion/24/what-is-good-place`? That seems a tad long...
The other option would be making the backend give the community id in the discussion but that seems like code smell.

Reply:
Do you mean putting this under the community heading ("Welcome to Amsterdam" in the design example)? But yeah that URL seems a little clunky.

Isn't the idea of this page is for people to access the discussion standalone?